docs/questions/query-builder/expressions/datetimeadd.md

```diff
@@ -44,26 +44,11 @@ You can use the [`between`](../expressions-list.md#between) or [`interval`](../e
 | Boolean                 | ❌                   |
 | JSON                    | ❌                   |
 
-This table uses `timestamp` and `datetime` interchangeably---just make sure that your dates and times aren't stored as string or a number data types in your database.
+This table uses `timestamp` and `datetime` interchangeably---just make sure that your dates and times aren't stored as strings or numbers in your database.
```

Comment:
We should probably link to the docs on casting to datetimes. https://www.metabase.com/docs/latest/data-modeling/metadata-editing#casting-to-a-specific-data-type